aiida/cmdline/params/options/commands/computer.py

```diff
@@ -45,13 +45,18 @@ def should_call_default_mpiprocs_per_machine(ctx):  # pylint: disable=invalid-na
     return job_resource_cls.accepts_default_mpiprocs_per_machine()
 
 
-LABEL = options.LABEL.clone(prompt='Computer label', cls=InteractiveOption, required=True)
+LABEL = options.LABEL.clone(
+    prompt='Computer label',
+    cls=InteractiveOption,
+    required=True,
+    help='Unique, human-readable label for this computer'
+)
 
 HOSTNAME = options.HOSTNAME.clone(
     prompt='Hostname',
     cls=InteractiveOption,
     required=True,
-    help='The fully qualified hostname of the computer; use "localhost" for local transports.',
+    help='The fully qualified hostname of the computer (e.g. daint.cscs.ch); use "localhost" for local transports.',
```

Comment:
@ltalirz?

Reply:
> Would the following work:
> 
> > use "localhost" for this machine

I think the original reason for the phrasing may be that it is difficult to find another term besides localhost that is unambiguous and easy to understand ("this machine" is certainly ambiguous in the context of setting up a new computer).

> This doesn't make any sense. You don't choose the hostname based on what transport you want to use.

Could you educate me then? I.e. have you come across any instance where 'localhost' is not the right choice for a computer configured for local transport?

Reply:
> I think the original reason for the phrasing may be that it is difficult to find another term besides localhost that is unambiguous and easy to understand ("this machine" is certainly ambiguous in the context of setting up a new computer).

That's fair enough. How about

> Use "localhost" if this is the same computer that AiiDA is running on


> > This doesn't make any sense. You don't choose the hostname based on what transport you want to use.
> 
> Could you educate me then? I.e. have you come across any instance where 'localhost' is not the right choice for a computer configured for local transport?

It is not so much the fact that the `localhost` should not always use `local` transport, but the way it is phrased, the direction of dependency is reversed. If you want to use the machine that AiiDA itself is running on as a computer, you should name it `localhost` and since it is the localhost, it should use the `local` transport. The selection of the hostname also comes before the transport.

Reply:
I'd be ok with

> Use "localhost" when setting up the computer that AiiDA is running on

Personally, I still find the original version short and succinct.